package.json

```diff
@@ -10,12 +10,16 @@
     "strapi": "strapi"
   },
   "dependencies": {
+    "3": "^2.1.0",
```

Comment:
What is the `3` and `-` packages for?

Reply:
I have no idea about it, just upgraded the marked package, let me take a look.

Reply:
@chungthuang, I redo it from the beginning by git clone from main branch, then install version upgrade. currently don't have the package of 3 and -.